src/java/main/org/apache/zookeeper/server/quorum/Follower.java

```diff
@@ -196,6 +224,10 @@ protected long getLastQueued() {
         return lastQueued;
     }
 
+    public Integer getSyncedObserverSize() {
+        return  om == null ? null : om.getNumActiveObservers();
```

Comment:
should we just return 0 if om is null? This would simplify caller code like https://github.com/apache/zookeeper/pull/628/files#diff-8c1d5f61ac728fda8f4b706ae4277364R369

Is there a need to differentiate between null om and none-null om with zero number of active observers?

Reply:
I was allowing the `synced_observers` stat to distinguish between "this follower CAN support observers, it just currently isn't" and "this follower CANNOT support observers (and so it isn't)" by outputting a value of 0 or by being omitted respectively.

It's a style choice. I'm not terribly attached to my current implementation and can change it if you think outputting `synced_observers` of 0 in both cases is more straightforward.

Reply:
making sense to me. I think current approach is good.